product-service/serverless.ts

```diff
@@ -0,0 +1,80 @@
+import type { AWS } from '@serverless/typescript';
+
+import { getProductById, getProductList } from '@functions';
+
+const serverlessConfiguration: AWS = {
+  service: 'product-service',
+  frameworkVersion: '2',
+  plugins: [
+    'serverless-esbuild',
+    'serverless-openapi-documentation',
+  ],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs14.x',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
+    },
+    lambdaHashingVersion: '20201221',
+    region: 'eu-west-1',
+  },
+  // import the function via paths
+  functions: { getProductList, getProductById },
+  package: { individually: true },
+  custom: {
+    esbuild: {
+      bundle: true,
+      minify: true,
+      sourcemap: true,
+      exclude: ['aws-sdk'],
+      target: 'node14',
+      define: { 'require.resolve': undefined },
+      platform: 'node',
+      concurrency: 10,
+    },
+    documentation: {
```

Comment:
[Minor]
It's better to move this section out of the global `serverless.ts` file to a separate module.

Reply:
Read this from docs. Decided to skip it this time due to the small size of the section.